packages/authhero/test/routes/auth-api/token.spec.ts

```
@@ -112,7 +112,7 @@ describe("token", () => {
 
   describe("authorization_code", () => {
     describe("authorization_code", () => {
-      it("should return an access token but no id token if the openid scope is missing", async () => {
+      it.only("should return an access token but no id token if the openid scope is missing", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)